documentation/tutorial.md

```diff
@@ -68,3 +75,762 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
+```shell
+/etc/mongodb/external-ca.crt  /etc/mongodb/external-cert.pem
+```
+
+As before, we will connect to MongoDB via the saved MongoDB URI. Connect using the saved URI and the following TLS options:
+```shell
+mongosh mongodb://$DB_USERNAME:$DB_PASSWORD@$HOST_IP/$DB_NAME?replicaSet=$REPL_SET_NAME --tls --tlsCAFile /etc/mongodb/external-ca.crt --tlsCertificateKeyFile /etc/mongodb/external-cert.pem
+```
+
+Congratulations, you've now connected to MongoDB with TLS. Now exit the MongoDB shell by typing:
+```shell
+exit
+```
+Now you should be back in the host of Charmed MongoDB (`mongodb/0`). To exit this host type:
+```shell
+exit
+```
+You should now be shell you started in where you can interact with Juju and LXD.
+
+### Disable TLS
+To disable TLS unrelate the two applications:
+```shell
+juju remove-relation mongodb tls-certificates-operator
+```
+
+## Next Steps
+In this tutorial we've successfully deployed MongoDB, added/removed replicas, added/removed users to/from the database, and even enabled and disabled TLS. You may now keep your Charmed MongoDB deployment running and write to the database or remove it entirely using the steps in [Remove Charmed MongoDB and Juju](#remove-charmed-mongodb-and-juju). If you're looking for what to do next you can:
+- Run [Charmed MongoDB on Kubernetes](https://github.com/canonical/mongodb-k8s-operator).
+- Check out our Charmed offerings of [PostGres](https://charmhub.io/postgresql?channel=edge) and [Kafka](https://charmhub.io/kafka?channel=edge).
+- Read about [High Availability Best Practices](https://canonical.com/blog/database-high-availability)
+- [Report](https://github.com/canonical/mongodb-operator/issues) any problems you encountered.
+- [Give us your feedback](https://chat.charmhub.io/charmhub/channels/data-platform).
+- [Contribute to the code base](https://github.com/canonical/mongodb-operator)
+
+## Remove Charmed MongoDB and Juju
+If you're done using Charmed MongoDB and Juju and would like to free up resources on your machine you can remove Charmed MongoDB and Juju. *Disclaimer when you remove Charmed MongoDB as shown below you will lose all the data in MongoDB. Further when you remove Juju as shown below you will lose access to any other applications you have hosted on Juju.*
+
+To remove Charmed MongoDB and the model it is hosted on run the command:
+```shell
+juju destroy-model tutorial --destroy-storage
+```
+
+Next step is to remove the juju controller. You can see all of the available controllers by entering `juju controllers`. To remove the controller enter:
+```shell
+juju destroy-controller overlord
```

Comment:
It didn't like me running this command: 

![image](https://user-images.githubusercontent.com/116022293/211790467-1d77cc63-7a8b-487a-aed4-ee954d42395c.png)

Reply:
good to know it seems like the command above that deleted the model didn't succeed. Adding force to the previous command will make this work so I will do just that 😉